content/en/blog/_posts/2023-09-20-happy-7th-birthday-kubeadm.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: blog
+title: 'Happy 7th birthday kubeadm!'
+date: 2023-09-20
```

Comment:
Is this the actual 7th birthday? I thought it would be from the v1.4 release day which was on a different date.

Reply:
Fixed to the 1.4 release date